### PR TITLE
TEMP namespace packages bug exploration

### PR DESCRIPTION
**Pull request by thundergolfer**
### Description 

Just putting this up to facilitate sharing.

### CODE REVIEW

File: rules_pip_test/main.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+import os
+import snowflake.connector
```
Comment (thundergolfer) [inline rules_pip_test/main.py:4]: should be able to `import` this, but can't

File: rules_pip_test/BUILD [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+]
+
+py_binary(
+    name = "main",
+    srcs = ["main.py"],
+    imports = ["."],
+    main = "main.py",
+    python_version = "PY3",
+    srcs_version = "PY3",
+    deps = PY_DEPS,
```
Comment (ali5h) [inline rules_pip_test/BUILD]: add `legacy_create_init = False`, I will try to fix this issue

File: WORKSPACE [hunk 3]
```diff
@@ -86,14 +86,35 @@ load("@io_bazel_rules_python//python:pip.bzl", "pip_import", "pip_repositories")
...
-# Load the pip_install symbol for my_deps, and create the dependencies repositories.
-load("@pip_dependencies//:requirements.bzl", "pip_install")
+load("@com_github_ali5h_rules_pip//:defs.bzl", "pip_import", "repositories")
+
+repositories()
+
+pip_import(
+   name = "rules_pip_deps",
+   requirements = "//tools/dependencies:python_requirements.txt",
+   python_version="3",
```
Comment (ali5h) [inline WORKSPACE]: add a minor version to get passed the `enum34` problem

File: WORKSPACE [hunk 4]
```diff
@@ -86,14 +86,35 @@ load("@io_bazel_rules_python//python:pip.bzl", "pip_import", "pip_repositories")
...
+## Load the pip_install symbol for my_deps, and create the dependencies repositories.
+#load("@pip_dependencies//:requirements.bzl", "pip_install")
+#
+#pip_install()
+
+## rules_pip
+
+http_archive(
+    name = "com_github_ali5h_rules_pip",
+    strip_prefix = "rules_pip-0.5.0",
```
Comment (ali5h) [inline WORKSPACE:103]: use version 1.0.0

Comment (thundergolfer) [reply WORKSPACE:103]: Thanks for following up on this, mate. Definitely helped me out. 👏

**Pull request closed without merging**